naresh/src/main/java/com/lftechnology/phpjava/ems/views/EmployeeView.java

```diff
@@ -0,0 +1,373 @@
+package com.lftechnology.phpjava.ems.views;
+
+import com.lftechnology.phpjava.ems.constants.Constant;
+import com.lftechnology.phpjava.ems.controllers.EmployeeController;
+import com.lftechnology.phpjava.ems.entities.Employee;
+import com.lftechnology.phpjava.ems.enums.Role;
+import com.lftechnology.phpjava.ems.utlis.CommonUtility;
+import com.lftechnology.phpjava.ems.utlis.ConsoleWriter;
+import com.lftechnology.phpjava.ems.utlis.Router;
+import com.lftechnology.phpjava.ems.utlis.UserInput;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * EmployeeView
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 16 2016
+ */
+public class EmployeeView implements ViewSignature<Object> {
+    public static CommonUtility commonUtility = new CommonUtility();
+    private EmployeeController employeeController = new EmployeeController();
+    private String action;
+
+    /**
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    @Override
+    public void render() {
+        switch (action) {
+            case Constant.POST_LOGIN_ACTION:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1151)